config/500-webhook-deployment.yaml

```diff
@@ -82,6 +82,23 @@ spec:
           seccompProfile:
             type: RuntimeDefault
 
+        readinessProbe:
+          periodSeconds: 1
+          httpGet:
+            scheme: HTTPS
+            port: 8443
+            httpHeaders:
+              - name: k-kubelet-probe
+                value: "webhook"
+        livenessProbe:
+          periodSeconds: 1
+          httpGet:
+            scheme: HTTPS
+            port: 8443
+            httpHeaders:
+              - name: k-kubelet-probe
+                value: "webhook"
```

Comment:
Should we have same values with serving's webhook (i.e. should not be same between readiness and liveness )? - https://github.com/knative/serving/blob/main/config/core/deployments/webhook.yaml#L122-L123

Here are the discussion:
* https://github.com/knative/serving/pull/9215
* https://github.com/knative/serving/issues/9077

Reply:
Yes I am aware of that was not sure to what extend it is a problem.